client.go

```diff
@@ -732,9 +734,36 @@ func (c *Client) logStderr(r io.Reader) {
 		line, err := bufR.ReadString('\n')
 		if line != "" {
 			c.config.Stderr.Write([]byte(line))
-
 			line = strings.TrimRightFunc(line, unicode.IsSpace)
-			c.logger.Named(filepath.Base(c.config.Cmd.Path)).Debug(line)
+
+			l := c.logger.Named(filepath.Base(c.config.Cmd.Path))
+			// If output is not JSON format, print directly as error
+			if !isJSON(line) {
```

Comment:
I ended up not needing to add anything to the handshake since this allows the output to be printed as-is on the host if hclog format (i.e. JSON formatted output) is not supported on the plugin.